jre_emul/Classes/com/google/j2objc/net/IosHttpURLConnection.java

```diff
@@ -545,6 +549,42 @@ private static IOException secureConnectionException(String description) {
       completionHandler(nil);
     }
   }]-*/
+  
+
+  protected List<Certificate> certificates = new ArrayList<Certificate>();
+  private static CertificateFactory certificateFactory = null;
```

Comment:
Remove static var and move certificateFactory to addToCertificateList(). CertificateFactory uses the Apache Fortress Engine to look up and instantiate the service, and caches it so it's fast to look up multiple times.